### DISCUSSION TITLE
Support float logical properties

### DISCUSSION THREAD

**Discussion by datlechin** [tailwindlabs/tailwindcss#10902] [Ideas]
Hey Tailwind CCSS team,

As a mentioned in v3.3 https://tailwindcss.com/blog/tailwindcss-v3-3#simplified-rtl-support-with-logical-properties

We have logical properties for text align, padding, margin, ...

But it doesn't included `float`, ex: `float-start`, `float-end`

**Comment by wongjn** [comment]
For what it's worth, the original PR #10166 did mention `float`:

> **This PR does not include support for the logical values of `clear` or `float`**, as browser support is currently poor. We can of course revisit this in the future and add them another day when support is better.

**Comment by thibaudcolas** [comment]
👋 browser support now feels good enough for flow-relative `float` and `clear`. I’ve opened #12469 to request support in Tailwind.
